cli/args/mod.rs

```diff
@@ -378,10 +380,16 @@ impl LintOptions {
       rules: Default::default(),
       files: FilePatterns::new_with_base(base),
       fix: false,
+      plugins: vec![],
+      dir_path: PathBuf::from("/"),
```

Comment:
Seems like a bug?

Reply:
I think we should store the plugins as fully resolved paths here?

Reply:
Done!